### PR TITLE
Adds some fun adminbus gear, the Solar Armor and Daybreak

### PR DESCRIPTION
**Pull request by ShadeAware**
## About The Pull Request
Adds the Solar Armor, an absolutely OP 90 melee/bullet/fire resistant armor set. Also adds the Daybreak, a spear-like weapon which deals 50 damage on normal attack and 150 on throw.

**Both of these items are adminspawn only.**

## Why It's Good For The Game
Funny Terraria references, fun Admin stuff.

## Changelog
:cl:
add: Solar Armor for Adminbus
add: Daybreak spear weapon for Adminbus
/:cl:

### CODE REVIEW

**Comment by H0gberg** [comment]
I will be using this to admin bus these dum tators, very cool mike

File: modular_skyrat/code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -703,3 +703,20 @@
...
+	righthand_file = 'modular_skyrat/icons/mob/inhands/weapons/melee_righthand.dmi'
+	icon_state = "daybreak"
+	item_state = "daybreak"
+	force = 50 
+	throwforce = 150
+	armour_penetration = 30
+	sharpness = IS_SHARP_ACCURATE
+	attack_verb = list("stabbed", "jabbed", "slashed")
+	hitsound = 'sound/weapons/daybreakslice.ogg'
+	w_class = WEIGHT_CLASS_BULKY
```
Comment (Gandalf2k15) [inline modular_skyrat/code/game/objects/items/weaponry.dm:722]: Aaaah, newline pls

File: modular_skyrat/code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -48,3 +48,18 @@
...
+	icon = 'modular_skyrat/icons/obj/clothing/suits.dmi'
+	mob_overlay_icon = 'modular_skyrat/icons/mob/clothing/suit.dmi'
+	icon_state = "solar_armor"
+	item_state = "solar_armor"
+	armor = list("melee" = 90, "bullet" = 90, "laser" = 90, "energy" = 90, "bomb" = 90, "bio" = 100, "rad" = 100, "fire" = 90, "acid" = 100)
+	allowed = list(/obj/item/clockwork/weapon/daybreak)
+	resistance_flags = INDESTRUCTIBLE | FIRE_PROOF | ACID_PROOF | LAVA_PROOF
+	blocks_shove_knockdown = TRUE
+	strip_delay = 80
+	equip_delay_other = 60
```
Comment (Gandalf2k15) [inline modular_skyrat/code/modules/clothing/suits/armor.dm:65]: Newline here too plssss

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]

**Comment by ShadeAware** [comment]
Ill work more on this later, I just can't be assed to fix the conflicts right now with all of the projects I have going on

**Pull request closed without merging**